src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Rectangle extends FigureAbstract implements Figure {
+    private double width;
+    private double height;
+    public Rectangle(Color color, double width, double heigth) {
+        super(color);
+        this.width = width;
+        this.height = heigth;
```

Comment:
There is a typo in the constructor parameter name 'heigth'. It should be 'height' to match the class field name and maintain consistency.